README.md

```diff
@@ -76,6 +76,13 @@ It also guarantees that the design is sound before code is written; a GitHub pul
 Typos and grammatical errors can go straight to a pull-request.
 When in doubt, start on the [mailing-list](#mailing-list).
 
+## Weekly Call
+
+The contributors and maintainers of all OCI projects have a weekly meeting Wednesdays at 10:00 AM (USA Pacific.)
+Everyone is welcome to participate via [UberConference web][UberConference] or audio-only: 646-494-8704 (no PIN needed.)
```

Comment:
You're missing the [reference-style link definition][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v0.5.0/README.md#L157